src/python/pants/notes/1.19.x.rst

```diff
@@ -0,0 +1,177 @@
+1.19.x Stable Releases
+======================
+This document describes releases leading up to the ``1.19.x`` ``stable`` series.
+
+1.19.0rc0 (29/07/2019)
```

Comment:
I think this is missing like 10 commits?

Reply:
I'm not sure, because iirc the 1.18.rc0 notes only included the changes from 1.18.dev<last> to the RC, and the range from the last dev release to this one is pretty narrow: https://pastebin.com/5dTSxrAZ

Reply:
I think your master may be a while out of date?
```
pants$ git log --oneline release_1.19.0.dev0..master
a5ddac3 Revert "use jdeps to run dep-usage task (#8093)"
d45af28 Fail for missing classpath entries with hermetic execution. (#8037)
adc9898 Properly manage the lifetime of Exiters in Daemon Runs (#7996)
96f6dd2 Deprecate ZincCompile task in favor of RscCompile (#8047)
23a8a2b Add support for scoverage (#8064)
536d216 Lower test timeouts in travis to get better debug output. (#8110)
a1b9eb9 Upgrade cloc to 1.80 and add error handling (#8111)
44c2741  Rewrite `test_git_hooks.py` to be hermetic (#8085)
b5d41da use jdeps to run dep-usage task (#8093)
0a45f40 Prepare 1.17.0rc2 and render 1.18.x. (#8114)
a996b57 Balance pytest test sharding (#8108)
1a1238e Record per-target compile workflow stats when using RscCompile (#8092)
89a6425 Parse zinc args and pass scalac options into scalafix (#8091)
5967b2f Loosen constraints on the jvm heap size for tests in extra_jvm_options (#8106)
4854194 Properly handle source roots and resources with V2 Pytest runner (#8063)
b76a9d2 add script to generate zinc native-images, with example usage (#8036)
81993f4 pants build committing publish data for push of org.pantsbuild#scoverage-report-generator_2.12;0.0.1 [ci skip]
b0877f8 Scoverage report generator (#8098)
e41ef3b [pantsd] Don't ignore the first watchman event for the daemon pid (#8096)
ec8e274 Add fs_util subcommand to list known directories (#8100)
0760fe8 [pantsd] Remove dead code around prefork graph warming (#8095)
570fab8 Report the stats version pants is using to the server. (#8086)
54d0d4e Use remote build execution to run several unit tests (#8051)
5fd255e Remove unused unit-test.sh script (#8083)
35be511 Update CONTRIBUTORS.md
74f280b Use direct import of version.py (#8078)
f9bb8d6 Refactor V2 PEX creation rules (#8080)
```

Reply:
OH shoot you're right, I need to rebase. Sorry :(

Reply:
Changes made :)